amplify/.gitignore

```diff
@@ -0,0 +1,2 @@
+.parcel-cache
```

Comment:
What do you think about starting a directory for node packages like pkgs/ or packages/ and putting amplify/ under it? We do this for clojure libs with deps/ and it helps reduce clutter at the top-level. With tldraw and libs, this will be the third node package in this mono repo

Reply:
It's a good idea! I have refactored the amplify/ to packages/ in this branch :)